partiql-tests-data/fail/syntax/primitives/time-constructor.ion

```diff
@@ -70,30 +70,6 @@
     },
 }
 
-{
```

Comment:
Any context for these tests being removed?

Reply:
Hmm...I thought I'd added to the summary but looks like I didn't save. I pasted the message in the corresponding commit there and added it to the following:

> Based on the https://github.com/partiql/partiql-lang-rust/pull/163 implementation of types and function parsing
we're ok with having the statements in the remove tests to go through
as we'll check them further later in static-analysis phase.

Reply:
Ok I see. So should these deleted tests be moved to the `fail/static-analysis/` folder?

Reply:
Yes, that's a good call—will do.